wca-regulations.md

```diff
@@ -131,7 +131,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3h2) Puzzles whose colored plastic is visible inside the puzzle (e.g. "stickerless" puzzles) are permitted. This does not include the following puzzles:
         - 3h2a) Puzzles with transparent parts that can reveal more information about the state of the puzzle. Exception: An overlay sticker or a logo (see [Regulation 3l](regulations:regulation:3l)).
     - 3h3) Any modifications to a puzzle that result in poor performance by a competitor are not grounds for additional attempts.
-    - 3h4) For Clock, custom "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
+    - 3h4) For Clock, the following modifications are permitted:
+        - 3h4a) Customs "inserts" (the same shape and size as the traditional paper inserts) are permitted, at the discretion of the WCA Delegate. The inserts must have a clear indication of 12 o'clock that matches the original inserts.
+        - 3h4b) Puzzle design customizations that do not give an unfair advantage may be permitted, at the discretion of the WCA Delegate (e.g. checkerboard pattern clocks, faces with a custom design).
```

Comment:
There are modifications that give advantages similar to these in nature. What comes to mind is putting tape on the sides of the puzzle to hold it together + make the clock stand up straight instead of tilting, similar to how the Angstrom will stand up straight due to straight edges. Would this regulation disallow these being used?

Reply:
I wouldn't consider that unfair advantage, but I recognize it can be ambiguous

Reply:
Maybe a clarification telling that this is not prohibited would be fine, do you agree?

Reply:
another example would be a top row of dials having a different plastic color than the other dials. There are a few different ways that similar things can be done so I believe that wording the guideline in a way that states that any modifications allowing competitors to more easily recognize puzzle orientation should not be considered an unfair advantage, a second guideline giving an specific example like the tape example that Jacob gave or the dial example that I gave could also be a helpful addition.